java/Yaroslav-Dzhur/src/com/zoolatech/lecture1/tasks/_5/Main.java

```diff
@@ -0,0 +1,28 @@
+package com.zoolatech.lecture1.tasks._5;
+
+/*
+Write a program that accepts two numbers and outputs the larger number.
+If numbers are equal - output “Numbers are equal”.
+*/
+
+import java.util.Scanner;
+
+public class Main {
+
+    public static void main(String[] args) {
+        Scanner scanner = new Scanner (System.in);
+
```

Comment:
Please pay attention to those extra lines you're adding: this might decrease the code readability (here we have one extra new line).